src/sys/fs/binfmt_misc.rs

```diff
@@ -6,14 +6,6 @@ pub fn enabled() -> ProcResult<bool> {
     Ok(val == "enabled")
 }
 
-struct BinFmtEntry {
```

Comment:
I know this struct is private and unused, but it'll soon be exposed.  Can you leave it in?